chef/site-cookbooks/wca/templates/gen-authorized-keys.sh.erb

```diff
@@ -14,6 +14,8 @@ fi
   --gh-org thewca \
   --gh-team "Software Team" \
   --keyfile /home/cubing/.ssh/authorized_keys
+# TODO GB enable once the software admin team has been populated on the website
```

Comment:
To clarify: this loads the SSH keys from GitHub, so we just need to populate the GitHub team, not the website one.

Reply:
Yes, that was just me not paying attention while copy/pasting my TODO comment from `user.rb`. Nice catch!